nav2_bringup/bringup/launch/nav2_bringup_launch.py

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) 2018 Intel Corporation
```

Comment:
Just curious, why introducing `bringup` directory?

Reply:
I was also curious why you are moving the files to a new directory here.

Reply:
As discussed offline, `spawn_turtlebot2` is a python package while the rest of bringup is cmake. These should be kept separate. I thought this was better than having the package on the top level.